CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## Review
+
+Do we still need this changelog? It seems we never tracked all changes that diverts from GDS since the fork. Do we want to update this or get rid of this file? It might be quite benefitial to keep a changelog automatically generated, but I don't see manual maintenance happening.
```

Comment:
This is definitely not up to date. We need to talk with PMs about what to do with this file.

Reply:
Can you discard this change?

Reply:
Removed.